Project.toml

```diff
@@ -28,7 +28,6 @@ GAPExt = "GAP"
 PolymakeExt = "Polymake"
 
 [compat]
-AbstractAlgebra = "^0.39.0"
```

Comment:
why remove the AA compat? that will block AutoMerge in the general registry

Reply:
Oops, thanks for catching it